license/README.txt

```diff
@@ -10,20 +10,43 @@ Documentation
 =============
 
 For the latest documentation see the Open Touryo web site at:
-https://github.com/OpenTouryoProject/OpenTouryoDocuments
+
+- OpenTouryoProject/OpenTouryoDocuments
+  https://github.com/OpenTouryoProject/OpenTouryoDocuments
+- FrontPage - Open 棟梁 Wiki
+  https://opentouryo.osscons.jp/
+
+Notes
+=============
+There is possibility that the library of OSS that not include
+source code automatically acquired from the Internet by the
+package manager like the NuGet. If so, must follow these licenses of the
+oss when using / distributing this OSS. Incidentally, the OSS that will
+be obtained by NuGet can be checked by referring to each package.cponfig.
 
 ==================================================
 
-�v���W�F�N�g�̐i���� 
+プロジェクトの進捗状況 
 ============== 
 
-Open����́A������Г�\�����[�V�����Y�iHitachi Solutions, Ltd.�j�̎Г�c�[���ł���B 
-Open����́A2014�N4���ɃI�[�v���\�[�X�Ƃ��Č�J���ꂽ�B
-�����āAOpen����Project�ɂ��ăt�H�[�N���ꂽ�B
-Web�T�C�g�Fhttp://www.osscons.jp/dotNetDevelopmentInfrastructure/
+Open棟梁は、株式会社日立ソリューションズ（Hitachi Solutions, Ltd.）の社内ツールである。 
```

Comment:
@daisukenishino 

Rewrite this sentence by referring [the file in OpenTouryo repository](https://github.com/OpenTouryoProject/OpenTouryo/blob/develop/license/README.txt).